src/dependencies/xz-5.6.2/CMakeLists.txt

```diff
@@ -167,6 +167,8 @@ if(NOT MSVC)
     endif()
 endif()
 
+OPTION(XZ_INSTALL OFF "Enable xz Install directives")
```

Comment:
Does `add_subdirectory(dependencies/xz-5.6.2 EXCLUDE_FROM_ALL)` not work? This way, we can exclude everything. Other dependencies should be able to be excluded in the same way.

Sorry for the late comment.